Change control socket filenames to maintain compatibility with
previous versions of this package.

--- a/src/bin/keactrl/kea-ctrl-agent.conf.pre
+++ b/src/bin/keactrl/kea-ctrl-agent.conf.pre
@@ -32,15 +32,15 @@
     "control-sockets": {
         "dhcp4": {
             "socket-type": "unix",
-            "socket-name": "kea4-ctrl-socket"
+            "socket-name": "kea-dhcp4-ctrl.sock"
         },
         "dhcp6": {
             "socket-type": "unix",
-            "socket-name": "kea6-ctrl-socket"
+            "socket-name": "kea-dhcp6-ctrl.sock"
         },
         "d2": {
             "socket-type": "unix",
-            "socket-name": "kea-ddns-ctrl-socket"
+            "socket-name": "kea-dhcp-ddns-ctrl.sock"
         }
     },
 
--- a/src/bin/keactrl/kea-dhcp-ddns.conf.pre
+++ b/src/bin/keactrl/kea-dhcp-ddns.conf.pre
@@ -23,7 +23,7 @@
   "port": 53001,
   "control-socket": {
       "socket-type": "unix",
-      "socket-name": "kea-ddns-ctrl-socket"
+      "socket-name": "kea-dhcp-ddns-ctrl.sock"
   },
   "tsig-keys": [],
   "forward-ddns" : {},
--- a/src/bin/keactrl/kea-dhcp4.conf.pre
+++ b/src/bin/keactrl/kea-dhcp4.conf.pre
@@ -49,7 +49,7 @@
     // more. For detailed description, see Sections 8.8, 16 and 15.
     "control-socket": {
         "socket-type": "unix",
-        "socket-name": "kea4-ctrl-socket"
+        "socket-name": "kea-dhcp4-ctrl.sock"
     },
 
     // Use Memfile lease database backend to store leases in a CSV file.
--- a/src/bin/keactrl/kea-dhcp6.conf.pre
+++ b/src/bin/keactrl/kea-dhcp6.conf.pre
@@ -43,7 +43,7 @@
     // description, see Sections 9.12, 16 and 15.
     "control-socket": {
         "socket-type": "unix",
-        "socket-name": "kea6-ctrl-socket"
+        "socket-name": "kea-dhcp6-ctrl.sock"
     },
 
     // Use Memfile lease database backend to store leases in a CSV file.
